tests/foreman/cli/test_contentview.py

```diff
@@ -5323,7 +5322,56 @@ def test_positive_arbitrary_file_repo_promotion(self):
         :expectedresults: Check arbitrary files from FR is available on
             environment
 
-        :CaseAutomation: notautomated
+        :CaseAutomation: automated
 
         :CaseLevel: Integration
         """
+        org = make_org({u'name': gen_alphanumeric()})
```

Comment:
```suggestion
        org = make_org(cached=True)
```
Make as above , by default `make_org` takes `u'name': gen_alphanumeric(6)`

Reply:
Same has been incorporated.